Makefile

```diff
@@ -129,6 +129,10 @@ $(INTERMEDIATEDIR)/plugins/MySQL.d $(INTERMEDIATEDIR)/plugins/MySQL.o: plugins/M
 	@mkdir -p $(dir $@)
 	$(CXX) $(CXXFLAGS) -MMD -MF $(INTERMEDIATEDIR)/plugins/MySQL.d -MT $(INTERMEDIATEDIR)/plugins/MySQL.o $(GIT_REVISION) -o $(INTERMEDIATEDIR)/plugins/MySQL.o -c plugins/MySQL.cpp
 
+$(INTERMEDIATEDIR)/plugins/MySQLUtils.d $(INTERMEDIATEDIR)/plugins/MySQLUtils.o: plugins/MySQLUtils.cpp
+	@mkdir -p $(dir $@)
+	$(CXX) $(CXXFLAGS) -MMD -MF $(INTERMEDIATEDIR)/plugins/MySQLUtils.d -MT $(INTERMEDIATEDIR)/plugins/MySQLUtils.o $(GIT_REVISION) -o $(INTERMEDIATEDIR)/plugins/MySQLUtils.o -c plugins/MySQLUtils.cpp
+
```

Comment:
@tylerkaraszewski is there a better way to go about this? not specifically the make code but in general with regards to the utils specific to a generic plugin. I think the other plugins, like Jobs, just keep all of the code in the plugin file itself as part of the plugin class or in libstuff. Not really sure what the 'best' answer is though